File: templates/Uwp/_composition/_shared/Feature.UnitTests.Core.SampleData.ContentGrid.MSTest._VB/Param_ProjectName.Core.Tests.MSTest/Tests_postaction.vb
﻿'{[{
Imports Param_RootNamespace.Core.Services
'}]}

Public Class Tests
    '^^
    '{[{

    ' TODO WTS: Remove or update this once your app is using real data and not the SampleDataService.
    ' This test serves only as a demonstration of testing functionality in the Core library.
    <TestMethod>
    Public Sub EnsureSampleDataServiceReturnsContentGridData()
        Dim actual = SampleDataService.GetContentGridData()

        Assert.AreNotEqual(0, actual.Count)
    End Sub
    '}]}
End Class

File: templates/Uwp/_composition/_shared/Feature.UnitTests.Core.SampleData.ImageGallery.NUnit/Param_ProjectName.Core.Tests.NUnit/Tests_postaction.cs
﻿//{[{
using Param_RootNamespace.Core.Services;
//}]}

namespace Param_RootNamespace.Core.Tests.NUnit
{
    public class Tests
    {
        //^^
        //{[{

        // TODO WTS: Remove or update this once your app is using real data and not the SampleDataService.
        // This test serves only as a demonstration of testing functionality in the Core library.
        [Test]
        public void EnsureSampleDataServiceReturnsGalleryData()
        {
            var actual = SampleDataService.GetGallerySampleData();

            Assert.AreNotEqual(0, actual.Count);
        }
        //}]}
    }
}
